app/controllers/concerns/load_settings.rb

```diff
@@ -18,6 +18,12 @@ def build_settings_defaults
         website_logo_url: '',
         website_url: ''
       })
+      SettingsObject.create(title: 'styles', values: {
+        css: ''
+      })
+      SettingsObject.create(title: 'scripts', values: {
+        javascript: ''
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.